Change Server/cmdline Tutorial to use new `dart` tool (#2757)
dart-lang/site-www: src/_tutorials/server/cmdline.md

## Running an app with the standalone Dart VM

> To run a command-line app, you need the Dart command (`dart run`),
> which comes when you [install the Dart SDK](/get-dart).

Reviewer: I'd like to tweak this a bit and change the SDK link:

```suggestion
To run a command-line app in the Dart VM, use `dart run`.
The `dart` command is part of the [Dart SDK](/tools/sdk).
```

Author: Done

You might notice that dcat depends on a package named **args**.
To get the args package, use the

> [dart pub package manager](/guides/packages).

Reviewer: It's still the pub package manager; only the command needs the `dart` preface.

```suggestion
[pub package manager](/guides/packages).
```

Author: Okay, thanks for the clarification.
